packages/gatsby-theme-api-docs/src/components/resource/method/parameters.js

```diff
@@ -14,6 +15,13 @@ import { DescriptionText } from '../../description';
 import Info from '../../info';
 import { typography } from '../../../design-system';
 
+const isRegex = (string) =>
+  string.charAt(0) === '/' && string.charAt(string.length - 1) === '/';
+
+const getExpressionInsideSlashes = (input) => {
+  return input.match(/\/([^;]*)\//);
+};
```

Comment:
I think you should not duplicate the code but re-use it.   Either from a generic utils package or by exporting them from the type component.